R/estimatr_lh_robust.R

```diff
@@ -66,60 +93,17 @@ lh_robust <- function(formula,
                       alpha = .05,
                       return_vcov = TRUE,
                       try_cholesky = FALSE,
-                      linearHypothesis = NULL,
-                      lh_term = NULL
+                      linearHypothesis = NULL
                       ) {
+    call <- match.call()
+    args <- as.list(call)
+    args[[1]] <- NULL
+    args$linearHypothesis <- NULL
+    model <- do.call(lm_robust, args)
```

Comment:
This is better, but check that NSE still works with weights, subset and add a test.

Reply:
This does fine with weights and subsets locally. Though it's not stable for tests and checks. Data and other arguments aren't found in the environment.

Copy-pasting lm_robust unless there are other suggestions

Reply:
Ah I see this comment now. I don't know how to solve this problem.

Reply:
> Data and other arguments aren't found in the environment.

That sounds like you need to adjust the envir / overscoping for eval.

Reply:
Also, an alternative solution would be to use `...` instead of manually capturing arguments.

Reply:
the `...` approach works great [see commit](https://github.com/DeclareDesign/estimatr/pull/281/commits/c04a5e53562d0af9d5cf258c0709eeb79a8f4f22).

 I was sure I had tried that already.